dockerfiles/base/scripts/base/startup_funcs.sh

```diff
@@ -403,12 +527,17 @@ start() {
   set -- "${@/\-\-debug/}"
   set -- "${@/\-\-offline/}"
   set -- "${@/\-\-trace/}"
+  set -- "${@/\-\-skip\:preflight/}"
+  set -- "${@/\-\-skip\:postflight/}"
+  set -- "${@/\-\-skip\:nightly/}"
+  set -- "${@/\-\-skip\:network/}"
+  set -- "${@/\-\-skip\:pull/}"
   
-    # The pre_init method is unique to each assembly. This method must be provided by 
+  # The cli_post_init method is unique to each assembly. This method must be provided by 
   # a custom CLI assembly in their container and can set global variables which are 
   # specific to that implementation of the CLI. This method must be called after
   # networking has been established and initial images downloaded.
-  cli_pre_init
+  cli_post_init
```

Comment:
I think here it's odd to have renamed cli_pre_init to cli_post_init

previous flow was
 ```
 cli_pre_init
 cli_init "$@"
```

so cli_pre_init is called before cli_init

with this change the flow is

```
cli_pre_init
init "$@"
cli_post_init
cli_init
```

so we see cli_post_init being called before cli_init which is not IMHO expected

- pre_init and post_init should be before/after init method
- cli_pre_init and cli_post_init should be before/after cli_init